Refreshing numel on a stride update is pointless.

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32116 Refreshing numel on a stride update is pointless.**
* #32114 Delete resize_dim()
* #32112 Expunge last two sites of resize_dim

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D19579875](https://our.internmc.facebook.com/intern/diff/D19579875)

ezyang committed:
Refreshing numel on a stride update is pointless.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit a8fd46ed:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=a8fd46ed071d915f7904c46b7f9272d6910532b5).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4221944) pytorch_xla_linux_xenial_py3_6_clang7_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=84391873) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=84391873))
<details>
<summary>
<code>Jan 13 21:03:47 RuntimeError: Failed to apply patch: /var/lib/jenkins/workspace/xla/torch_patches/X10-clip_grad.diff</code>
</summary>

```
Jan 13 21:03:47 + XDIR=/var/lib/jenkins/workspace/xla/scripts/.. 
Jan 13 21:03:47 + PTDIR=/var/lib/jenkins/workspace/xla/scripts/../.. 
Jan 13 21:03:47 + python /var/lib/jenkins/workspace/xla/scripts/cond_patch.py /var/lib/jenkins/workspace/xla/scripts/../torch_patches /var/lib/jenkins/workspace/xla/scripts/../.. 
Jan 13 21:03:47 1 out of 1 hunk FAILED 
Jan 13 21:03:47 Applying patch file: /var/lib/jenkins/workspace/xla/torch_patches/X10-clip_grad.diff 
Jan 13 21:03:47 Traceback (most recent call last): 
Jan 13 21:03:47   File "/var/lib/jenkins/workspace/xla/scripts/cond_patch.py", line 67, in <module> 
Jan 13 21:03:47     patch_repo(args) 
Jan 13 21:03:47   File "/var/lib/jenkins/workspace/xla/scripts/cond_patch.py", line 49, in patch_repo 
Jan 13 21:03:47     raise RuntimeError('Failed to apply patch: {}'.format(ppath)) 
Jan 13 21:03:47 RuntimeError: Failed to apply patch: /var/lib/jenkins/workspace/xla/torch_patches/X10-clip_grad.diff 
Jan 13 21:03:47 + cleanup 
Jan 13 21:03:47 + retcode=1 
Jan 13 21:03:47 + set +x 
Jan 13 21:03:47 =================== sccache compilation log =================== 
Jan 13 21:03:47 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 13 21:03:47 Compile requests                 1 
Jan 13 21:03:47 Compile requests executed        0 
Jan 13 21:03:47 Cache hits                       0 
Jan 13 21:03:47 Cache misses                     0 
Jan 13 21:03:47 Cache timeouts                   0 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

ezyang committed:
Update on "Refreshing numel on a stride update is pointless."


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Refreshing numel on a stride update is pointless."


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Refreshing numel on a stride update is pointless."


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Refreshing numel on a stride update is pointless."


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D19579875](https://our.internmc.facebook.com/intern/diff/D19579875)

[ghstack-poisoned]
